ElementInternalsType/explainer.md

```diff
@@ -234,6 +251,180 @@ customElements.define('custom-button', CustomButton);
 
 ## Alternatives considered
 
+### ElementInternals feature decomposition approach
+An alternative approach focuses on decomposing native element behaviors into granular functionalities that can be individually exposed through `ElementInternals`. This approach also builds on the existing pattern established by form-associated custom elements ([FACEs](https://html.spec.whatwg.org/dev/custom-elements.html#form-associated-custom-elements)) and accessibility semantics ([ARIAMixin](https://www.w3.org/TR/wai-aria-1.2/#ARIAMixin)), where specific capabilities are exposed as discrete APIs that web developers can combine as needed.
+

```

Comment:
adding a quick example here (like the structure of the main approach) should help provide better context for people reading the follow paragraph, eg
```
class CustomButton extends HTMLElement {
    static canUseCommandInvocation= true;
}
customElements.define('custom-button', CustomButton);
```